### PR TITLE
add laranuxt and oneconcern

### PR DESCRIPTION
**Pull request by acidjazz**

### CODE REVIEW

**Comment by acidjazz** [comment]
@manniL done!

File: README.md [hunk 1]
```diff
@@ -308,6 +307,7 @@ Nuxt.js is a framework for creating Universal Vue.js Applications.
 - [exeteres/electron-nuxt-ts](https://github.com/exeteres/electron-nuxt-ts) - Another Electron starter template with separation of the renderer process and the main process that fully written in TypeScript.
 - [muhibbudins/nuxt-coreui](https://github.com/muhibbudins/nuxt-coreui) - Unofficial Nuxt + CoreUI project, free to use boilerplate for every need.
 - [Tarrask/sails-template](https://github.com/Tarrask/sails-template) - Starter template for Nuxt.js with Sails.js.
+- [Laranuxt boilerplate](https://github.com/acidjazz/laranuxt) - by [@acidjazz](https://github.com/acidjazz) - Laravel + Nuxt.js using the official proxy jmodule
```
Comment (pi0) [inline README.md]: `proxy module` :)

File: README.md [hunk 2]
```diff
@@ -218,6 +218,7 @@ Nuxt.js is a framework for creating Universal Vue.js Applications.
 - [El Cable Amarillo](https://www.elcableamarillo.cc/) - Open source project for the use of programming and robotics in educational centers using software tools and free hardware.
 - [Zip Stores](https://zip.co/stores/) - Zip Pay and Zip Money Stores Directory. Zip is simple, fair and transparent credit and payment solutions that improve lives every day. We allow people to own the things they love now, and pay later, interest free.
 - [61 Financial News](https://news.61financial.com.au) - Australia financial news platform for Chinese readers, powered by Nuxt.js with best practices and well-tuned performance.
+- [OneConcern](https://oneconcern.com) [source](https://github.com/acidjazz/oneconcern) - Artifical Intelligence platform for Disasters.
```
Comment (manniL) [inline README.md]:
```suggestion
- [OneConcern](https://oneconcern.com) ([source](https://github.com/acidjazz/oneconcern)) - Artifical Intelligence platform for Disasters.
```

File: README.md [hunk 3]
```diff
@@ -306,6 +307,7 @@ Nuxt.js is a framework for creating Universal Vue.js Applications.
 - [exeteres/electron-nuxt-ts](https://github.com/exeteres/electron-nuxt-ts) - Another Electron starter template with separation of the renderer process and the main process that fully written in TypeScript.
 - [muhibbudins/nuxt-coreui](https://github.com/muhibbudins/nuxt-coreui) - Unofficial Nuxt + CoreUI project, free to use boilerplate for every need.
 - [Tarrask/sails-template](https://github.com/Tarrask/sails-template) - Starter template for Nuxt.js with Sails.js.
+- [Laranuxt boilerplate](https://github.com/acidjazz/laranuxt) - by [@acidjazz](https://github.com/acidjazz) - Laravel + Nuxt.js using the official proxy module
```
Comment (manniL) [inline README.md]:
```suggestion
- [Laranuxt boilerplate](https://github.com/acidjazz/laranuxt) - Laravel + Nuxt.js using the official proxy module
```

Minor suggestion: As the author is clear through the owner of the repo, it doesn't need to be explicitly added. At least I haven't seen anyone else doing that :thinking:

**Pull request merged**